frontend/src/components/playlist.js

```diff
@@ -1,19 +1,254 @@
 import React, { Component } from 'react';
-import logo from '../logo.svg';
+import { Input, Button, Select } from 'antd';
+import { Container, Row, Col } from 'reactstrap';
+import 'antd/dist/antd.css';
+import '../styles/playlist.css'
 import '../App.css';
+import Embedded from './embedded'
+
+
+const { Search, TextArea } = Input;
+const { Option } = Select;
+
+let accessToken = ''
+let Spotify = require('spotify-web-api-js');
+let spotifyApi = new Spotify();
+spotifyApi.setAccessToken(accessToken);
 
 class Playlist extends Component {
+  constructor(props) {
+    super(props);
+    this.state={
+      user: {
+          id: '',
+          playlistId: '',
+          iframe:''
+      },
+      artist: {
+          name: '',
+          id: ''
+      },
+      playlistName: 'Playlist App',
+      playlistDescription: 'Created with Playlist App',
+      selectedArtists: [], 
+      searchedArtists: [],
+      topTracksIds: [],
+      displayPlaylist: 'none',
+      random: 0
+    }
+  }
+
+  componentDidMount() {
+      this.getUser();
+  }
+
+  sliceUserId = (url) => {
+      let string = url.split(/[/?]/);
+      return string[5]
+  }
+
+  getUser = () => {
+      spotifyApi.getUserPlaylists().then(
+        (data) => {
+            this.setState({
+                user: { id : this.sliceUserId(data.href)}
+            })
+        },
+        (err) => {
+        console.error(err);
+        }
+    );
+  }
+
+  //this is for showing the artist names before creating playlist
+  getArtist = (artist) => {
+    spotifyApi.searchArtists(artist).then(
+
+        (data) => {
+            if(data !== undefined) {
+                this.setState({ // change to index 1 if query is same
+                    searchedArtists: this.state.searchedArtists.concat(data.artists.items[0].name), // index is 0 because most popular query's index is 0 
+                    selectedArtists: this.state.selectedArtists.concat(data.artists.items[0].name)
+                })
+            }
+        },
+
+        (err) => {
+            console.log(err)
```

Comment:
Can you create a ticket to display an error message to the user if the artist they have entered is not found.